pkg/controller/ttl_manager.go

```diff
@@ -320,6 +321,18 @@ func resolveTtlStatus(ns *coreapi.Namespace, processPods func() (bool, time.Time
 		status.active = active
 		status.logger = status.logger.WithField("active", active)
 		if !lastTransitionTime.IsZero() {
```

Comment:
I don't have enough knowledge of the code, but is it possible for `lastTransitionTime` to be zero while active annotation is present? Would we still want use the active annotation + softttl in that case?

Reply:
no, because soft TTL only applies when the namespace is not active (when last transition time is nonzero)